packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/quanticRadioButtonsGroup.js

```diff
@@ -0,0 +1,160 @@
+import {LightningElement, api, track} from 'lwc';
+
+/**
+ * The `QuanticRadioButtonsGroupOption` type defines the shape of an option
+ * that can be passed to the `QuanticRadioButtonsGroup` component.
+ * @typedef {Object} QuanticRadioButtonsGroupOption
+ * @property {String} value The value of the option.
+ * @property {String} [label] The label of the option.
+ * @property {String} [iconName] The name of the icon to be displayed.
+ * @property {String} [tooltip] The tooltip of the option.
+ */
+
+export default class QuanticRadioButtonsGroup extends LightningElement {
+  /**
+   * The value of the label for the Radio Buttons Group.
+   * @api
+   * @type {String}
+   */
+  @api legend = '';
+  /**
+   * The list of options to be displayed.
+   * @api
+   * @type {QuanticRadioButtonsGroupOption[]}
+   */
+  @api options;
+  /**
+   * If the labels should be hidden or visible.
+   * @api
+   * @type {Boolean}
+   */
+  @api hideLabels;
+  /**
+   * The value of the selected radio button inside the Radio Buttons Group.
+   * @api
+   * @type {String}
+   */
+  @api get value() {
+    return this._value;
+  }
+  set value(value) {
+    this._value = value;
+  }
+  /**
+   * The size of the icon.
+   * @api
+   * @type {'xx-small' | 'x-small' | 'small' | 'medium' | 'large'}
+   */
+  @api
+  get iconSize() {
+    return this._iconSize;
+  }
+  set iconSize(value) {
+    if (['xx-small', 'x-small', 'small', 'medium', 'large'].includes(value)) {
+      this._iconSize = value;
+    }
+  }
+
+  @track _value;
+
+  /** @type {'xx-small' | 'x-small' | 'small' | 'medium' | 'large'} */
+  _iconSize = 'xx-small';
+
+  get radioButtonElements() {
+    return this.template.querySelectorAll('input');
+  }
+
+  get transformedOptions() {
+    const {options, value} = this;
+    if (Array.isArray(options)) {
+      return options.map((option, index) => ({
+        label: option.label,
+        value: option.value,
+        iconName: option.iconName,
+        iconVariant: value === option.value ? 'inverse' : '',
```

Comment:
After renaming, we would need to update the occurrences of `value` to be like the following:
```suggestion
        iconVariant: selectedValue === option.value ? 'inverse' : '',
```